server/modules/service-discovery/consul/consulCatalog.js

```diff
@@ -39,7 +39,17 @@ function getAllNodes(environment) {
 
 function getNode(environment, nodeName) {
   assert(nodeName, 'nodeName is required');
-  return executeConsul(environment, clientInstance => clientInstance.catalog.node.services(nodeName));
+  return executeConsul(environment, clientInstance => clientInstance.catalog.node.services(nodeName))
+    .then((nodes) => {
+      if (!nodes || !nodes.Services) {
```

Comment:
Not sure it's necessary. These are properties on an object returned from the consul SDK, so there's no ambiguity.